README.md

```diff
@@ -1,7 +1,15 @@
 # Nice(TSY) Cypher System Add-ons
-![](https://img.shields.io/badge/Foundry-v0.8.9-informational?style=flat-square)
-![GitHub Latest Release](https://img.shields.io/github/release/NiceTSY/nice-cypher-add-ons?style=flat-square)
-![](https://img.shields.io/badge/license-BSD--3--Clause-green?link=https://opensource.org/licenses/BSD-3-Clause&style=flat-square)
+[![ko-fi](https://img.shields.io/badge/Ko--Fi-farling-success)](https://ko-fi.com/farling)
```

Comment:
We need to discuss about those links, I will not link any for myself but as you worked on the repo I think we could squeeze those somewhere if you want.

Reply:
Those links aren't required, you can reinstate your original version.

Reply:
Reverted the changes in my main